### DISCUSSION TITLE
Is it possible to add a Badge?

### DISCUSSION THREAD

**Discussion by imJack6** [AvaloniaCommunity/Material.Avalonia#346] [Ideas]
rt, I want to use badge, or are there any other examples?

**Comment by Wilhelm-Lee** [comment]
I develop with imJack6, we're a team.

To be brief, it is something we'd like to have, a badge that attaches on image, and display any given content, from string to integer, mostly anything.  Now, we had tried to write ourselves, but it turns out that it requires modifications in library, not just something we can do within our own applications.

So, is there any possibility we can have this feature added, or is there any existing code we can sample from?

**Comment by SKProCH** [comment]
Currently there is no badge control available, but I'd try to develop one.

**Reply by imJack6** [reply]
Thanks :>
